core/src/main/scala/kanaloa/util/JavaDurationConverters.scala

```diff
@@ -0,0 +1,14 @@
+package kanaloa.util
```

Comment:
Moved. As an anecdote, I added the same class to `akka.util` 
https://github.com/akka/akka/pull/18357/files#r40411548